content/docs/introduction/comparison.md

```diff
@@ -244,43 +244,39 @@ blackbox probing is sufficient.
 If you want to do whitebox monitoring, or have a dynamic or cloud based
 environment, then Prometheus is a good choice.
 
+From https://prometheus.io/docs/introduction/comparison/
```

Comment:
Remove this (as it's a link to the current page)

Reply:
@lucperkins Yup, just saw that